### PR TITLE
INTMDB-488: Analytics node tier new features

### PR DESCRIPTION
**Pull request by martinstibbe**
## Description

Analytics node tier new features

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

File: mongodbatlas/resource_mongodbatlas_cloud_backup_snapshot_export_job.go [hunk 1]
```diff
@@ -278,17 +278,17 @@ func resourceMongoDBAtlasCloudBackupSnapshotExportJobImportState(ctx context.Con
...
 	if err != nil {
-		return nil, fmt.Errorf("couldn't import snapshot export job %s in project %s and cluster %s, error: %s", exportJobID, projectID, clusterName, err)
+		return nil, fmt.Errorf("couldn't import snapshot export job %s in project %s and cluster %s, error: %s", exportID, projectID, clusterName, err)
 	}
 
 	d.SetId(encodeStateID(map[string]string{
 		"project_id":    projectID,
 		"cluster_name":  clusterName,
-		"export_job_id": exportJobID,
+		"export_job_id": exportID,
```
Comment (matt-condon) [inline mongodbatlas/resource_mongodbatlas_cloud_backup_snapshot_export_job.go:291]: Is it possible to update `export_job_id` references to `export_id` to align with the updated property name `exportId`?

Comment (Zuhairahmed) [reply mongodbatlas/resource_mongodbatlas_cloud_backup_snapshot_export_job.go:291]: @matt-condon this is from old PR which has already been merged [here](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/976). We need to re-base to fix older edits from coming into newer PRs. Would ignore this one for now.

**Review by Zuhairahmed — APPROVED** [review]
LGTM! Great work here @martinstibbe

**Review by matt-condon — APPROVED** [review]
LGTM

**Pull request merged**